File: code/lucky_sum.py
def lucky_sum(array):
	sum = 0
	for n in array:
		if n == 13:
			break
		else:
			sum = sum + n
	return sum

print(lucky_sum([2,13,2]))
print(lucky_sum([13,2,3]))
print(lucky_sum([2,4,13]))

File: code/luckySum.java
public class luckySum {
	
	public static void main(String args[]){
		 int a,b,c,sum;
		 sum=luckySum(13,2,3);
		 System.out.println("Sum is " + sum);
		 sum=luckySum(1,13,3);
		 System.out.println("Sum is " + sum);
		 sum=luckySum(1,2,13);
		 System.out.println("Sum is " + sum);
		 sum=luckySum(1,2,3);
		 System.out.println("Sum is " + sum);
	 }
	
	 public static int luckySum (Integer... numbers) {
	      int sum = 0;
	      for(int i=0; i <numbers.length; i++) {
	        if(numbers[i] == 13) {
	          break;
	        }
	        sum = sum + numbers[i];
	      }
	      return sum;
	 }
} 
